libvips/colour/LabQ2sRGB.c

```diff
@@ -227,9 +227,19 @@ vips_col_sRGB2scRGB_16( int r, int g, int b, float *R, float *G, float *B )
 int
 vips_col_scRGB2XYZ( float R, float G, float B, float *X, float *Y, float *Z )
 {
-	*X = SCALE * 0.4124 * R + SCALE * 0.3576 * G + SCALE * 0.18056 * B;
-	*Y = SCALE * 0.2126 * R + SCALE * 0.7152 * G + SCALE * 0.07220 * B;
-	*Z = SCALE * 0.0193 * R + SCALE * 0.1192 * G + SCALE * 0.9505 * B;
+	R *= SCALE;
+	G *= SCALE;
+	B *= SCALE;
+
+	*X = 0.4124 * R +
+		0.3576 * G +
+		0.1805 * B;
```

Comment:
I was going to suggest that 0.18056 be rounded up to 0.1806, but all the evidence I can find suggests that 0.1805 is the "right" value anyway, so :shipit: